Assets/Scripts/Limb.cs

```diff
@@ -0,0 +1,55 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Limb : MonoBehaviour
+{
+    [SerializeField] private int speed = 1300;
+
+    [SerializeField] private Rigidbody2D rb;
+
+    [SerializeField] private float angleLeft;
```

Comment:
to differentiate left/right limb control
for angle that will be twisted when press left/right